src/components/timecard/select-timecard-period-type/SelectTimecardPeriodType.jsx

```diff
@@ -48,15 +35,21 @@ const SelectTimecardPeriodType = () => {
       <form
         className="select-timecard-period-type"
         onSubmit={handleSubmit((data) => {
-          setTimecardData({ ...timecardData, timePeriodType: data[radioName] });
+          setTimeEntries([
+            ...timeEntries,
+            {
+              timePeriodType: data[radioName],
+              timePeriodTypeId: '00000000-0000-0000-0000-000000000001',
```

Comment:
Are we still hard coding shift type to shift even though we’re bringing them in from the database now?

Reply:
I've added in some code to retrieve the timePeriodTypeId

Reply:
Lovely 👍 Although I’ve noticed that the unit tests for `SelectTimecardPeriodType` don’t cover the successful submit case. Would you agree that now’s a good time to add that in? Especially now there’s some logic in there for grabbing the selected timeperiodtype

Reply:
I've added a test to cover the submit now

Reply:
For the record, Laurie explored implementing this test by mocking the `handleSubmit` function that we get back from `useForm` and we agreed that this was super gnarly. We agreed to skip it for now.